javascript/Javascript.md

```diff
@@ -0,0 +1,149 @@
+﻿# JavaScript
+
+## Projected time
+
+About 30-45 minutes
+Lesson: 10 min 
+Guided Practice:15 min
+Check for Understanding:20 min
+
+## Prerequisites
+
+There is not touch prerequisites for learning JavaScript. Here are some of the prerequisite for learning JavaScript:
+
++ Basic programming knowledge. Knowledge of core Java sufficient.
+
++ You should have knowledge of HTML.Check [HTML tutorials here.](https://www.w3schools.com/html/)
```

Comment:
```suggestion
+ You should have knowledge of HTML. Check [HTML tutorials here.](https://www.w3schools.com/html/)
```
And we already have `web/html.md` for HTML so I don't think that we should link some external link for the same.